box.json

```diff
@@ -38,7 +38,8 @@
     "scripts":{
         "format":"cfformat run models/**/*.cfc,tests/specs/**/*.cfc --overwrite",
         "format:check":"cfformat check models/**/*.cfc,tests/specs/**/*.cfc --verbose",
-        "generateAPIDocs":"rm .tmp --recurse --force && docbox generate mapping=cbauth excludes=tests|ModuleConfig strategy-outputDir=.tmp/apidocs strategy-projectTitle=cbauth"
+        "generateAPIDocs":"rm .tmp --recurse --force && docbox generate mapping=cbauth excludes=tests|ModuleConfig strategy-outputDir=.tmp/apidocs strategy-projectTitle=cbauth",
+        "install:2021":"cfpm install zip:2021.0.02.328618,debugger:2021.0.02.328618"
```

Comment:
Is this going to break later to install generically `adobe@2021` as the server but specifically `2021.0.02.328618` as the version for the packages?

I realize this came up because the release happened before ForgeBox was updated, but that might be our edge case.

Reply:
Well, Adobe's PM is really simplistic and just terrible in that sense.  I had to do this before update 5 was released. It might bite us later.  But for now I am reverting back.